Classes/Issues/IssuesViewController.swift

```diff
@@ -549,14 +548,10 @@ final class IssuesViewController:
     func didSelectReply(to sectionController: IssueCommentSectionController, commentModel: IssueCommentModel) {
         setMessageView(hidden: false, animated: true)
         messageView.textView.becomeFirstResponder()
-        messageView.text = ">"
-            + getCommentUntilNewLine(from: commentModel.rawMarkdown)
-            + "\n\n"
-            + "@"
-            + commentModel.details.login
+        let quote = getCommentUntilNewLine(from: commentModel.rawMarkdown)
+        messageView.text = ">\(quote)\n\n@\(commentModel.details.login)"
```

Comment:
What about making this a multiline String literal to keep it a bit more readable?

```swift
let quote = """
>\(quote)
\n\n
@\(commentModel.details.login)
"""
```

Reply:
Hm, this supports only the first line of a comment? :/

Reply:
Haha I replied to the wrong cell. Check above 😄